### ISSUE TITLE
Syctemctl --user start scylla-server Failing

### ISSUE THREAD

**Issue by dreamerHarshit** [scylladb/scylla#8145]
Trying to install Scylla-unified installer(Scylla 4.3) on RHEL8, Although installation is done successfully but start server command is failing.

Scylla version (or git commit hash): Scylla 4.3
Cluster size: 1 node
OS (RHEL/CentOS/Ubuntu/AWS AMI): RHEL8

*Hardware details (for performance issues)*          Delete if unneeded
Platform (physical/VM/cloud instance type/docker): VM
Hardware: cores=4 memory=32GB

command = `systemctl --user start scylla-server`

error = `Job for scylla-server.service failed because of fatal signal was delivered causing the control process to dump core. See "systemctl status scylla-server.service" and "journalctl -xe" for details. `

**Comment by slivne** [comment]
@dreamerHarshit can you please provide full logs from the server.

**Comment by dreamerHarshit** [comment]
Can't share complete logs because of security concerns, please checking the following if this can help

`scylla-housekeeping-restart.timer. Job scylla-housekeeping-restart.timer/start failed with result 'dependency'`

**Comment by slivne** [comment]
did you enable houskeeping / version check at installation ?

If so can you please try to disable it and see if that solves your issue

**Comment by dreamerHarshit** [comment]
No, I have not enabled it, I just followed the steps mentioned on our website for unified-installer, so not sure how to enable/disable it.

**Comment by dreamerHarshit** [comment]
@slivne any update on this?

**Comment by slivne** [comment]
@dreamerHarshit can you provide uname -a the exact info from the VM

@penberg / @syuu1228 any idea whats the issue - can we try and run a test locally with non root ... and see it works for us with that version.

another thing that is not clear if he did not enable version check why is houskeeping attempting to run ?

**Comment by dreamerHarshit** [comment]
sure.. one more thing we are not using GCP or AWS, we are using our internal cloud.

**Comment by dreamerHarshit** [comment]
Linux xxxxxxx-xxx.xx.xx.com 4.18.0-193.19.1.el8_2.x86_64 #1 SMP Wed Aug 26 15:29:02 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux @slivne

**Comment by syuu1228** [comment]
I tested scylla-unified-package-4.3.1.0.20210226.74941f67e.tar.gz on RHEL-8.3.0_HVM-20201031-x86_64-0-Hourly2-GP2 (EC2), 
it just worked fine with the procedure described in unified-installer document.
I think it's not possible to resolve the issue without error log, we at least need `systemctl --user status --no-pager -l scylla-server.service`.

One possible reason is not yet executed iotune.
It occur when we input "no" to this prompt:
```
Do you want IOTune to study your disks IO profile and adapt Scylla to it? (*WARNING* Saying NO here means the node will not boot in production mode unless you configure the I/O Subsystem manually!)
Yes - let iotune study my disk(s). Note that this action will take a few minutes. No - skip this step.
[YES/no]
```
If it is the reason, we can re-run scylla_setup script and type "yes" to the question.
Or, even typed "yes" to the question, iotune may fail to complete running for some reasons, then scylla-server won't able to start. (It can fail when disk space < 10GB or filesystem is not supported, etc)

**Comment by yaronkaikov** [comment]
@syuu1228 Is this relevant? if not let's close it

**Comment by avikivity** [comment]
Closing as stale.
